### DISCUSSION TITLE
매크로 실행 취소

### DISCUSSION THREAD

**Discussion by hky0102** [kimytsc/covid-rest-vaccine-macro#61] [Q&A]
1. 매크로 실행되어 예약 시도중인데 예약 시도를 중단하려면 어떻게 해야하나요 ?
2. 예약 성공하지 못하고 당일 마감이되면 매크로 실행을 취소하고 다음날 다시 매크로 실행해야 하나요 ?

**Comment by kimytsc** [comment]
> 1. 매크로 실행되어 예약 시도중인데 예약 시도를 중단하려면 어떻게 해야하나요 ?

새로고침 또는 창을 끄시면 됩니다.

> 2. 예약 성공하지 못하고 당일 마감이되면 매크로 실행을 취소하고 다음날 다시 매크로 실행해야 하나요 ?

대부분 6시경에 병원들이 문을 닫는 것으로 생각되어, 종료하였다가 다음날 다시 매크로를 실행하시면 좋을 것 같습니다.

**Reply by hky0102** [reply]
감사합니다. 하나만 더 여쭙겠습니다.
카카오와 네이버 예약 시도 매크로창을 함께 띄어놔도 될까요 ?

**Reply by kimytsc** [reply]
네, 카카오와 네이버를 같이 띄워놓으셔도 됩니다.
